### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by harshithakolipaka**
## Description

Added docstrings to skimage.color .This updates the issue #6761 

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
  `doc/release/release_dev.rst`.
- There is a bot to help automate backporting a PR to an older branch. For
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.

### CODE REVIEW

File: skimage/color/__init__.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+""" Utilities for color conversion, color labeling, and color difference calculations."""
```
Comment (jni) [inline skimage/color/__init__.py:1]:
```suggestion
"""Utilities for color conversion, color labeling, and color difference calculations."""
```

**Review by jni — APPROVED** [review]
Thank you @harshithakolipaka! I made a small formatting change (removing initial whitespace in the docstring), but otherwise this looks good to me.

File: skimage/color/__init__.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+"""Utilities for color conversion, color labeling, and color difference calculations."""
```
Comment (lagru) [inline skimage/color/__init__.py:1]:
```suggestion
"""Utilities for color conversion, color labeling, and color difference
calculations.
"""
```
Thanks @harshithakolipaka. Another small formatting tweak by me. We usually try to keep our line length within 79 characters according to [PEP8](https://peps.python.org/pep-0008/#maximum-line-length) (see also our [style guide](https://scikit-image.org/docs/dev/contribute.html#stylistic-guidelines)). I don't think our automation here checks this currently? But I thought it useful to point out. :)

File: skimage/color/__init__.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
+"""Utilities for color conversion, color labeling, and color difference
```
Comment (mkcor) [inline skimage/color/__init__.py:1]: I think that "color space conversion" would be clearer (and more accurate) than "color conversion" -- feel free to submit a follow-up PR @harshithakolipaka 

/cc @lagru

**Pull request merged**